websites/K/Kuroiru/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "629851641579044874",
+    "name": "Anant"
+  },
+  "service": "Kuroiru",
+  "description": {
+    "en": "Cross-site anime tracker. Track anime across multiple resources and stay updated."
+  },
+  "url": "kuroiru.co",
+  "version": "1.0.0",
+  "logo": "https://i.ibb.co/1fZY8jWs/icon-512x512.png",
+  "thumbnail": "https://i.ibb.co/htR9MxZ/letter-small.png",
```

Comment:
Thumbnail should be a promotional image/banner, or a screenshot of the homepage. not the logo